core/src/main/resources/error/error-classes.json

```
@@ -1432,6 +1432,12 @@
     },
     "sqlState" : "0A000"
   },
+  "UNSUPPORTED_EXPR_FOR_OPERATOR" : {
+    "message" : [
+      "Rewrite the query to avoid window functions, aggregate functions, and generator functions in the WHERE clause.",
```

Usually, we state the problem first of all, and then provide suggestions. Could you change the error format to:
```suggestion
      "A query operator contains one or more unsupported expressions. Consider to rewrite it to avoid window functions, aggregate functions, and generator functions in the WHERE clause.",
```

---

Thanks for improving this.